server/src/immich/controllers/person.controller.ts

```diff
@@ -26,8 +29,16 @@ export class PersonController {
   constructor(private service: PersonService) {}
 
   @Get()
-  getAllPeople(@AuthUser() authUser: AuthUserDto): Promise<PersonResponseDto[]> {
-    return this.service.getAll(authUser);
+  getAllPeople(
+    @AuthUser() authUser: AuthUserDto,
+    @Query(new ValidationPipe({ transform: true })) withHidden: PersonSearchDto,
```

Comment:
The `new ValidationPipe(...)` isn't actually needed here, since the class has `@UseValidation`.